src/main.dart

```diff
@@ -1,5 +1,183 @@
-import 'core/note_app.dart';
+import 'dart:io';
 
```

Comment:
It's considered a best practice to spread your classes across different files. Currently, you have all your code in one file, which can become challenging to read and manage as your codebase grows.